src/components/PropertiesList.tsx

```diff
@@ -0,0 +1,40 @@
+import PropertyDetails from "./PropertyDetails";
+
```

Comment:
Its  a good idea to keep components in its own folder. You can undo this change.